fontforge/mm.c

```diff
@@ -1110,8 +1110,8 @@ return( false );
 		} else if ( !mm->apple && !KernsMatch(sf->glyphs[i],mm->instances[j]->glyphs[i])) {
 		    if ( complain ) {
 			FVChangeGID( sf->fv,i);
-			ff_post_error(_("Bad Multiple Master Font"),_("The glyph %1$.30s in font %2$.30s has a different set of kern pairs than in %3$.30s"),
-				"vertical", sf->glyphs[i]->name,sf->fontname, mm->instances[j]->fontname);
+			ff_post_error(_("Bad Multiple Master Font"),_("The glyph vertical in font %2$.30s has a different set of kern pairs than in %3$.30s"),
```

Comment:
This string looks buggy at the first place - note that it used to have 3 parameters and 4 arguments. I think `"vertical"` should be just removed, the parameter `%1$.30s` is about glyph name, and it should be kept.

Reply:
Thank you so much! Didn't notice that.